spring-boot-dixit/src/main/java/tw/wally/dixit/DixitApplicationRunner.java

```diff
@@ -38,9 +40,11 @@ public void run(ApplicationArguments args) {
     }
 
     private Void registerService(RetryContext retryContext) throws RestClientException {
+        log.info("Dixit is registering itself to the Lobby. (Retry={})", retryContext.getRetryCount());
         Registration registration = new Registration(name, serviceHost, MIN_NUMBER_OF_PLAYERS, MAX_NUMBER_OF_PLAYERS);
         registration.addOption(new Registration.Option("winningScore", "range", 25, 35, 5));
         lobbyServiceDriver.registerService(registration);
+        log.info("Dixit registered itself to the Lobby successfully.");
```

Comment:
NIT: Can we use an aspect to log some information for debugging?

Reply:
No need for that, this is just a runner....